JungleMinds/JM_Assuline
# Changed font files with licensed ones

#### What does this PR do?

- [X] Implemented new font files
- [X] Edited config file and related css
- [X] Added disclaimer from the font company

#### How should this be tested?

`yarn` to install dependencies, `yarn start` to start development server. Go to `localhost:8000` and check the font

#### Any background context you want to provide?

#### What are the relevant tickets?

#### Definition of Done (remove if not applicable):
